### PR TITLE
BIG UPDATE! 

### PR DESCRIPTION
**Pull request by velotoxicsounds**
Hello

I have fixed the following points:
- Points display works again
- Telegram Bot integration [Also transmits score via Telegram]
- General element bug fixed
- Advanced login screen fixed

I have planned the following:
Rework code, human activity
All languages are supported because Bot works via XPath.

### CODE REVIEW

**Comment by koderian** [comment]
wow, thanks!

**Comment by koderian** [comment]
Mobile search and Edge searches dont seem to be functioning for me on your master branch @velotoxicsounds

**Comment by coollbv** [comment]
For some reason the middle daily quiz always requires log in and when the log in is clicked I get 400 Bad Request error. Anyone else?

File: Dockerfile_no_telegram [hunk 1]
```diff
@@ -2,17 +2,19 @@ FROM python:3.7
...
 RUN apt-get install -y google-chrome-stable
+run apt-get install -y sudo
 
 COPY . /rewards_bot
 WORKDIR /rewards_bot
 RUN pip install --no-cache-dir -r requirements.txt
 
 # Run the command on container startup
-CMD python3 /rewards_bot/ms_rewards.py --headless --all
+CMD if [ -e "verify" ] ; then telegram-send -g "Telegram already configured, start Bot. Please Wait" && python3 /rewards_bot/ms_rewards.py --headless --all  ; else touch verify && sudo telegram-send --configure --global-config && telegram-send --configure && telegram-send -g "This is a test message: Success, Initialize System Startup.... Please wait" && python3 /rewards_bot/ms_rewards.py --headless --all ; fi
```
Comment (kowshika-n) [inline Dockerfile_no_telegram:20]: This single command line seems too complicated. Please simplify for readability and maintenance.

File: ms_rewards.py [hunk 2]
```diff
@@ -782,40 +771,40 @@ def get_point_total(pc=False, mobile=False, log=False):
...
     # returns None if pc search not found
     # pcsearch = browser.find_element_by_class_name('pcsearch')
     # if pcsearch.location_once_scrolled_into_view is None:  # property causes pc search to be scrolled into area
     #     return False
 
     try:
+        time.sleep(3)
         current_point_total = list(map(
-            int, browser.find_element_by_class_name('credits2').text.split(' of ')))[0]
+            int, browser.find_element_by_css_selector('#userBanner > mee-banner > div > div > div > div.info-columns > div:nth-child(3) > mee-banner-slot-4 > mee-rewards-user-status-item > mee-rewards-user-status-dailypoint > div > div > div > div > div > p.bold.number.margin-top-1 > mee-rewards-counter-animation > span > b').text))[0]
```
Comment (kowshika-n) [inline ms_rewards.py:801]: Big CSS locators Iike this are very unstable and require frequent maintenance.
Please update an xpath or a better locator.

Comment (velotoxicsounds) [reply ms_rewards.py:801]: I adjusted. I create Git Pull right away. Future requests will only be made via direct ID allocation or XPATH. Now use the Katalon Recorder :) Makes work much :)

Comment (kowshika-n) [reply ms_rewards.py:801]: XPATHs like this are also unstable and will need frequent maintenance. Please check this [link](https://devhints.io/xpath) to get a better understanding of writing better XPATH.

**Comment by velotoxicsounds** [comment]
The http request error is currently normal. Has something to do with Microsoft's new login. I have just fixed the general login problem as well as the problem that no points are being distributed. Set a new push

**Pull request closed without merging**